public/js/cohortStudent.js

```diff
@@ -2,50 +2,32 @@ const addStd = document.getElementById('addStd');
 const studentName = document.getElementById('studentName');
 const gitHubUserName = document.getElementById('gitHubUserName');
 const addStudentButton = document.getElementById('addStudentButton');
-const loading = document.getElementById('loading');
 const Url = window.location.href;
 const splitUrl = Url.split('/');
 const cohortId = splitUrl[splitUrl.length - 2];
 const deleteStudentButton = document.querySelectorAll('.delete');
 
+
 const showAddStudentDiv = () => {
   addStd.classList.toggle('sectionAddstd--visible');
 };
 
-
 addStudentButton.addEventListener('click', (e) => {
   e.preventDefault();
 
-  const studentNameValue = studentName.value;
-  const gitHubUserNameValue = gitHubUserName.value;
-  if (
-    studentNameValue.trim() !== 0
-    && gitHubUserNameValue.trim() !== 0
-  ) {
+  studentNameValue = studentName.value;
+  gitHubUserNameValue = gitHubUserName.value;
+  if (studentNameValue.trim() !== 0 && gitHubUserNameValue.trim() !== 0) {
```

Comment:
So, `studentName` is a dom element (most probably an input), then you take it's value (most probably a string), then compare this string with a number? Are you sure this ever works?

Reply:
I edited it to studentNameValue.trim().length 
it' a mistake